### ISSUE TITLE
Wrong `main` entry in `package.json`

### ISSUE THREAD

**Issue by Harasz** [nats-io/nats.ws#129]
Hi,
I found a problem using your library, it's look like there is a wrong `main` entry file in  `package.json`.

Current: 
```
{
  "name": "nats.ws",
  "version": "1.7.0",
  "description": "WebSocket NATS client",
  "main": "./cjs/nats.cjs",
  "module": "./esm/nats.js",
  "exports": {
    ".": {
      "require": "./cjs/nats.js",
      "import": "./esm/nats.js"
    }
  },
....
```

Expected: 
```
{
  "name": "nats.ws",
  "version": "1.7.0",
  "description": "WebSocket NATS client",
  "main": "./cjs/nats.js", <-- CHANGE
  "module": "./esm/nats.js",
  "exports": {
    ".": {
      "require": "./cjs/nats.js",
      "import": "./esm/nats.js"
    }
  },
....
```

**Comment by aricart** [comment]
@Harasz thank you!

**Comment by Harasz** [comment]
Thanks for quick response :smile:

**Comment by aricart** [comment]
@Harasz I published v1.7.1 to npm. Thanks for noticing this.
